base/sparse/cholmod.jl

```diff
@@ -952,11 +952,11 @@ convert(::Type{Sparse}, A::Symmetric{Float64,SparseMatrixCSC{Float64,SuiteSparse
     Sparse(A.data, A.uplo == 'L' ? -1 : 1)
 convert(::Type{Sparse}, A::Hermitian{Tv,SparseMatrixCSC{Tv,SuiteSparse_long}}) where {Tv<:VTypes} =
     Sparse(A.data, A.uplo == 'L' ? -1 : 1)
-function convert{Ti<:ITypes}(::Type{Sparse},
+function convert(::Type{Sparse},
     A::Union{SparseMatrixCSC{BigFloat,Ti},
              Symmetric{BigFloat,SparseMatrixCSC{BigFloat,Ti}},
              Hermitian{Complex{BigFloat},SparseMatrixCSC{Complex{BigFloat},Ti}}},
-    args...)
+    args...) where Ti<:ITypes
```

Comment:
Perhaps indent the second and last lines of this signature further, so that they do not lie at the same indentation level as the method body?